src/main/java/wtf/g4s8/examples/spaxos/Proposer.java

```diff
@@ -120,7 +120,7 @@ private QuorumPrepared(final Proposal prop, T value, List<? extends Acceptor<T>>
         @Override
         public synchronized void promise(Proposal prop) {
             if (!done) {
-                Log.d("promise %s, cnt=%d", prop, this.cnt);
+                Log.d("promise %s, cnt=%d", prop, this.cnt + 1);
```

Comment:
@stepanvalyavskiy why we print `cnt + 1` but never increament it actually? (it seems to be printed `1` always)

Reply:
It increments on line 125. But idk why it's incremented after if block.